pkg/resourceinterpreter/customized/declarative/configurable.go

```diff
@@ -77,115 +78,136 @@ func (c *ConfigurableInterpreter) HookEnabled(kind schema.GroupVersionKind, oper
 	case configv1alpha1.InterpreterOperationReviseReplica:
 		script = accessor.GetReplicaRevisionLuaScript()
 	}
-	return len(script) > 0
+	return len(script) > 0, nil
 }
 
 // GetReplicas returns the desired replicas of the object as well as the requirements of each replica.
-func (c *ConfigurableInterpreter) GetReplicas(object *unstructured.Unstructured) (replicas int32, requires *workv1alpha2.ReplicaRequirements, enabled bool, err error) {
```

Comment:
The return value does not need to be modified; a declaration at the return point is sufficient.